<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSpinBox" name="SampleRate">
      <property name="maximum">
       <number>9999999</number>
      </property>
      <property name="value">
       <number>48000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="ChannelCount">
      <property name="value">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="SampleSize">
      <property name="value">
       <number>16</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Codec">
      <property name="text">
       <string>audio/pcm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ByteOrder">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>BigEndian</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LittleEndian</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="SampleType">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>Unknown</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SignedInt</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>UnSignedInt</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Float</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="convert">
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="playDirectly">
      <property name="text">
       <string>Play directly</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
